### PR TITLE
Bug 1610417 Quick Installer known issue

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1610417

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :rocket:

File: release_notes/ocp_3_7_release_notes.adoc [hunk 1]
```diff
@@ -2936,6 +2936,10 @@ duplicates and some are misleading in that they match wrong keys in the
 linux-audit dictionary.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1496176[*BZ#1496176*])
 
+*  NFS provisioning of the Quick Installer should be limited to hosts that are
```
Comment (kalexand-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: I'd replace the first sentence with "Limit NFS provisioning of the Quick Installer to hosts that are part of the cluster."

**Review by kalexand-rh — COMMENTED** [review]
One suggestion to avoid "should," but otherwise LGTM

**Comment by bmcelvee** [comment]
Thanks, @kalexand-rh! :space_invader:

**Pull request merged**